tests/test_catalogue/test_commands.py

```diff
@@ -430,7 +492,7 @@ def test_delete_400__not_cancelled_download_requests(self):
         assert CatalogueItem.objects.all().count() == 2
         assert response.status_code == 400
         assert response.json() == {
-            '@access': {'account_id': 25},
+            '@access': {'account_id': 28},
```

Comment:
this is changing dynamically so you can also use: `{'account_id': self.account.id}` and it will always pass